webhook/resources/backup/mutator.go

```diff
@@ -85,6 +86,25 @@ func (b *backupMutator) Create(request *admission.Request, newObj runtime.Object
 		patchOps = append(patchOps, fmt.Sprintf(`{"op": "replace", "path": "/spec/backupMode", "value": "%s"}`, string(longhorn.BackupModeIncremental)))
 	}
 
+	volume, err := b.ds.GetVolumeRO(volumeName)
+	if err != nil {
+		err := errors.Wrapf(err, "failed to get the volume %v of backup %v", volumeName, backup.Name)
+		return nil, werror.NewInvalidError(err.Error(), "")
+	}
```

Comment:
If the cluster is brand new and without any volumes, the check will prohibit the backup creation.